Inception/ClientApp/src/App/Models.ts

```diff
@@ -0,0 +1,14 @@
+export class SiteTestResult
```

Comment:
Extract in separate files with corresponding names

Reply:
+